README.md

```diff
@@ -171,6 +171,14 @@ You can run every example with zero setup as an "ANONYMOUS" Neptune user (no reg
 | Using Neptune in training jobs with custom Docker containers | | [![neptune]](https://app.neptune.ai/common/showroom/e/SHOW-29006) | [![github]](integrations-and-supported-tools/sagemaker/custom-docker-container/) |
 | Using Neptune in training jobs with PyTorch Estimator | | [![neptune]](https://app.neptune.ai/common/showroom/e/SHOW-29007) | [![github]](integrations-and-supported-tools/sagemaker/pytorch/) |
 
+
+### Azure ML
+
+| | Docs | Neptune | GitHub |
+| ----------- | :---: | :---: | :------: |
+| Using Neptune to track Azure ML jobs | [![docs]]( ) | [![neptune]](https://app.neptune.ai/o/common/org/project-time-series-forecasting/runs/details?viewId=standard-view&detailsTab=dashboard&dashboardId=988c17ae-0e46-41b1-b252-ca4fa975f344&shortId=TSF-824&type=run) | [![github]](integrations-and-supported-tools/azure-ml/) |
```

Comment:
Lets move it to the Integrations and Supported tools table instead of a separate table